### ISSUE TITLE
[Feature Request]  Rich Text - Drag and Drop Support

### ISSUE THREAD

**Issue by Ahmyo** [HappenApps/Quiver#995]
I want to use Quiver exclusively for Plain and Rich text including images.  I'd like to drag RTF and RTFD docs directly from my desktop to the Quiver Inbox.  Wonderful.

**Comment by ylian** [comment]
Good idea. Will add support for dropping files onto the app or notebooks.

**Comment by Ahmyo** [comment]
Fabulous!  Quiver is fun to use -- not only for programming languages. Rich text is alive and well for so many users.

**Comment by Ahmyo** [comment]
Almost a year has passed.  I still wait for more support for Rich Text, both import and export.  Is there anyone out there that can do a RTFD export script for Quiver?
